test/deltacache.js

```diff
@@ -86,8 +86,12 @@ describe('deltacache', () => {
     })
   })
 
-  after(() => {
-    serverP.then(server => server.stop())
+  after(done => {
+    serverP.then(server => {
+      server.stop().then(() => {
+        done()
+      })
+    })
```

Comment:
```
after(done => {
  serverP
     .then( server => server.stop())
     .then(done)
})
```
would be more promisy.

Reply:
I tried that, but stop passed the server to resolve(), so that gets passed to done. Which mocha does not like.